packages/ra-core/src/i18n/TranslationMessages.ts

```diff
@@ -0,0 +1,141 @@
+interface StringMap {
+    [key: string]: StringMap | string | undefined;
+}
+
+export interface TranslationMessages extends StringMap {
+    ra?: {
```

Comment:
Please remove all the question marks - we want all translations to contain at least these keys.